src/any_agent/tracing/attributes.py

```diff
@@ -0,0 +1,28 @@
+"""Constants exported for convenience to access span attributes.
```

Comment:
No strong opinion here but just offering up that this may be a candidate to be a `class GenAI(str, Enum)` instead of a collection of constants.

This might make it a more logical collection instead of generic constants like `OUTPUT` which might get confusing to people reading the code.

Reply:
Since all these is defined in a separate module, would it be ok for you if I just update the imports and docs to use something like:

```
from any_agent.tracing import attributes as span_attrs
...
    span_attrs.OUTPUT: "foo" 
```

I don't really see what would the benefit of adding a separate class or Enum

Reply:
Yep fine with me 👍